dev/ProgressRing/ProgressRing.cpp

```diff
@@ -74,7 +74,6 @@ void ProgressRing::OnApplyTemplate()
 
 void ProgressRing::OnSizeChanged(const winrt::IInspectable&, const winrt::IInspectable&)
 {
-    ApplyLottieAnimation();
```

Comment:
>ApplyLottieAnimation(); [](start = 4, length = 23)

Why isn't this needed anymore? If you change the size of the ProgressRing, while the indeterminate animation is playing, does the animation update appropraitely?

Reply:
LottiePlayer has the property `Stretch="fill"` set so that the animation scales along with the width. In the future, Size and StrokeWidth will also be a public, bindable property (like Foreground/Background) so that it updates without needing to re-apply the animation through `ApplyLottieAnimation()`.